src/pages/view/CacheableViewDashboard.js

```diff
@@ -15,29 +16,32 @@ import {
     sGetDashboardById,
     sGetDashboardsSortedByStarred,
 } from '../../reducers/dashboards.js'
+import { sGetSelectedId } from '../../reducers/selected.js'
 import ViewDashboard from './ViewDashboard.js'
 
 const CacheableViewDashboard = ({
-    id,
-    dashboardsLoaded,
+    clearSelectedDashboard,
     dashboardsIsEmpty,
+    dashboardsLoaded,
+    id,
+    selectedId,
 }) => {
-    const [dashboardsBarExpanded, setDashboardsBarExpanded] = useState(false)
     const { currentUser } = useCachedDataQuery()
 
+    useEffect(() => {
```

Comment:
What user scenario does this address?

Reply:
Before this code was in place, when a user would choose "Close dashboard" in the "action menu" dropdown, the selected dashboard would remain in the redux state. And now that we have this effect in place the redux state is cleared. I'm not 100% sure if this fix was also required to address a problem in the UI. I seem to remember that in some cases we would see the wrong information message. But in any case it does address a problem in the REDUX state, so good to keep anyway.

Reply:
Ok, can you please note this in the PR description as well? When this PR is merged you can copy all the relevant info in the PR description into the squash commit message.